config/routes.php

```diff
@@ -4464,6 +4464,18 @@
         ],
         'constraint' => [],
     ],
+    [
+        'name' => 'dar/templates',
+        'method' => 'get',
+        'path' => '/dar/templates/count/{field}',
+        'methodController' => 'DataAccessTemplateController@count',
+        'namespaceController' => 'App\Http\Controllers\Api\V1',
+        'middleware' => [
+            'jwt.verify',
+            'check.access:permissions,data-access-template.read',
+        ],
+        'constraint' => [],
```

Comment:
I guess the `field` is string.
Can you add in `constraint` ?

Reply:
Yes, it is a `string`. Constraint added